app/controllers/rubcop_test_controller.rb

```diff
@@ -0,0 +1,17 @@
+class RubocopController  < ApplicationController
```

Comment:
__[development][rubocop]__
* Style/Documentation: Missing top-level class documentation comment.
* Style/ExtraSpacing: Unnecessary spacing detected.
* Style/SpaceAroundOperators: Operator `<` should be surrounded by a single space. [:ref](https://github.com/bbatsov/ruby-style-guide#spaces-operators)